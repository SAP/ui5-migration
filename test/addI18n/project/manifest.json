{
  "_version": "1.8.0",
  "sap.app": {
    "id": "sap.f.cardsVisualTests",
    "type": "application",
    "i18n": "i18n/i18n.properties",
    "title": "{{appTitle}}",
    "description": "{{appDescription}}",
    "applicationVersion": {
      "version": "1.0.0"
    }
  },
  "sap.ui": {
    "technology": "UI5",
    "deviceTypes": {
      "desktop": true,
      "tablet": true,
      "phone": true
    }
  },
  "sap.ui5": {
    "services": {
      "UserRecent": {
        "factoryName": "cardsVisualTests.service.UserRecentFactory"
      },
      "Navigation": {
        "factoryName": "cardsVisualTests.service.SampleNavigationFactory"
      }
    },
    "dependencies": {
      "minUI5Version": "1.30.0",
      "libs": {
        "sap.ui.core": {},
        "sap.m": {},
        "sap.ui.integration": {
          "lazy": true
        },
        "sap.f": {},
        "sap.ui.layout": {
          "lazy": true
        },
        "sap.ui.codeeditor": {
          "lazy": true
        }
      }
    },
    "rootView": {
      "viewName": "sap.f.cardsVisualTests.view.App",
      "type": "XML",
      "async": true,
      "id": "app"
    },
    "models": {
      "i18n": {
        "type": "sap.ui.model.resource.ResourceModel",
        "settings": {
          "bundleName": "sap.f.cardsVisualTests.i18n.i18n"
        },
        "preload": true
      }
    },
    "routing": {
      "config": {
        "routerClass": "sap.m.routing.Router",
        "viewType": "XML",
        "viewPath": "sap.f.cardsVisualTests.view",
        "controlId": "app",
        "controlAggregation": "pages",
        "transition": "slide",
        "bypassed": {
          "target": "notFound"
        },
        "async": true
      },
      "routes": [
        {
          "pattern": "",
          "name": "appMain",
          "target": "main"
        },
        {
          "pattern": "grid",
          "name": "grid",
          "target": "grid"
        },
        {
          "pattern": "zflow",
          "name": "zflow",
          "target": "zflow"
        },
        {
          "pattern": "flexibleHeight",
          "name": "flexibleHeight",
          "target": "flexibleHeight"
        },
        {
          "pattern": "splitter",
          "name": "splitter",
          "target": "splitter"
        },
        {
          "pattern": "nolayout",
          "name": "nolayout",
          "target": "nolayout"
        },
        {
          "pattern": "listContent",
          "name": "listContent",
          "target": "listContent"
        },
        {
          "pattern": "tableContent",
          "name": "tableContent",
          "target": "tableContent"
        },
        {
          "pattern": "objectContent",
          "name": "objectContent",
          "target": "objectContent"
        },
        {
          "pattern": "adaptiveContent",
          "name": "adaptiveContent",
          "target": "adaptiveContent"
        },
        {
          "pattern": "flp",
          "name": "flp",
          "target": "flp"
        }, {
          "pattern": "numericHeader",
          "name": "numericHeader",
          "target": "numericHeader"
        },
        {
          "pattern": "analyticalContent",
          "name": "analyticalContent",
          "target": "analyticalContent"
        },
        {
          "pattern": "manifest",
          "name": "manifest",
          "target": "manifest"
        },
        {
          "pattern": "customContent",
          "name": "customContent",
          "target": "customContent"
        },
        {
          "pattern": "gridContainer",
          "name": "gridContainer",
          "target": "gridContainer"
        },
        {
          "pattern": "services",
          "name": "services",
          "target": "services"
        },
        {
          "pattern": "minHeight",
          "name": "minHeight",
          "target": "minHeight"
        }
      ],
      "targets": {
        "main": {
          "viewId": "main",
          "viewName": "Main",
          "viewLevel": 1
        },
        "grid": {
          "viewId": "grid",
          "viewName": "Grid",
          "viewLevel": 1
        },
        "zflow": {
          "viewId": "zflow",
          "viewName": "Zflow",
          "viewLevel": 1
        },
        "flexibleHeight": {
          "viewId": "flexibleHeight",
          "viewName": "FlexibleHeight",
          "viewLevel": 1
        },
        "splitter": {
          "viewId": "splitter",
          "viewName": "Splitter",
          "viewLevel": 1
        },
        "nolayout": {
          "viewId": "nolayout",
          "viewName": "NoLayout",
          "viewLevel": 1
        },
        "notFound": {
          "viewId": "notFound",
          "viewName": "NotFound",
          "transition": "show"
        },
        "listContent": {
          "viewId": "listContent",
          "viewName": "ListContent",
          "viewLevel": 1
        },
        "tableContent": {
          "viewId": "tableContent",
          "viewName": "TableContent",
          "viewLevel": 1
        },
        "objectContent": {
          "viewId": "objectContent",
          "viewName": "ObjectContent",
          "viewLevel": 1
        },
        "adaptiveContent": {
          "viewId": "adaptiveContent",
          "viewName": "AdaptiveContent",
          "viewLevel": 1
        },
        "analyticalContent": {
          "viewId": "analyticalContent",
          "viewName": "AnalyticalContent",
          "viewLevel": 1
        },
        "flp": {
          "viewId": "flp",
          "viewName": "Flp",
          "viewLevel": 1
        },
        "numericHeader": {
          "viewId": "numericHeader",
          "viewName": "NumericHeader",
          "viewLevel": 1
        },
        "manifest": {
          "viewId": "manifest",
          "viewName": "Manifest",
          "viewLevel": 1
        },
        "customContent": {
          "viewId": "custom",
          "viewName": "Custom",
          "viewLevel": 1
        },
        "gridContainer": {
          "viewId": "GridContainer",
          "viewName": "GridContainer",
          "viewLevel": 1
        },
        "services": {
          "viewId": "services",
          "viewName": "Services",
          "viewLevel": 1
        },
        "minHeight": {
          "viewId": "minHeight",
          "viewName": "MinHeight",
          "viewLevel": 1
        }
      }
    }
  }
}