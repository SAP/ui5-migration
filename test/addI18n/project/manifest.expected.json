{
  "_version": "1.21.0",
  "sap.app": {
    "id": "supi.theTests",
    "type": "application",
    "i18n": {
      "bundleUrl": "i18n/i18n.properties",
      "supportedLocales": [
        ""
      ],
      "fallbackLocale": ""
    },
    "title": "{{appTitle}}",
    "description": "{{appDescription}}",
    "applicationVersion": {
      "version": "1.0.0"
    }
  },
  "sap.ui": {
    "technology": "UI5",
    "deviceTypes": {
      "desktop": true,
      "tablet": true,
      "phone": true
    }
  },
  "sap.ui5": {
    "dependencies": {
      "minUI5Version": "1.30.0",
      "libs": {
        "sap.ui.core": {},
        "sap.m": {},
        "sap.ui.integration": {
          "lazy": true
        },
        "sap.f": {},
        "sap.ui.layout": {
          "lazy": true
        },
        "sap.ui.codeeditor": {
          "lazy": true
        }
      }
    },
    "rootView": {
      "viewName": "supi.theTests.view.App",
      "type": "XML",
      "async": true,
      "id": "app"
    },
    "models": {
      "i18n": {
        "type": "sap.ui.model.resource.ResourceModel",
        "settings": {
          "bundleName": "supi.theTests.i18n.i18n",
          "supportedLocales": [
            ""
          ],
          "fallbackLocale": ""
        },
        "preload": true
      }
    }
  }
}